src/backend/config/KsnipConfig.cpp

```diff
@@ -580,6 +580,19 @@ void KsnipConfig::setSnippingAreaPositionAndSizeInfoEnabled(bool enabled)
 	saveValue(KsnipConfigOptions::snippingAreaPositionAndSizeInfoEnabledString(), enabled);
 }
 
+bool KsnipConfig::autoShowMainWindowEnabled() const
```

Comment:
Maybe change the name to something that reflects that this affects the mainWindow behavior after taking screenshots?
Something like `showMainWindowAfterTakingScreenshot` ?

Reply:
Just commenting on the first entry, affects the other entries too.